doc/changes.md

```diff
@@ -1,5 +1,113 @@
 # Changelog
 
+## Release v0.14.0 (2024/12/31)
+
+[![Digital Object Identifier for PyGMT v0.14.0](https://zenodo.org/badge/DOI/10.5281/zenodo.14535921.svg)](https://doi.org/10.5281/zenodo.14535921)
+
+### Highlights
+
+*
```

Comment:
Any other highlights to add?

```suggestion
* 🎉 Fourteenth minor release of PyGMT 🎉
```

Reply:
Committed in 71bc440d8.

Reply:
Maybe the new `hline` and `vline` methods and the new functions to load more GMT remote datasets.

Reply:
Yes, see https://github.com/GenericMappingTools/pygmt/pull/3730#discussion_r1899271041